packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -193,6 +193,9 @@ export abstract class FluidDataStoreContext extends TypedEventEmitter<IFluidData
     private _disposed = false;
     public get disposed() { return this._disposed; }
 
+    private _tombstoned = false;
```

Comment:
Please add a comment desribing what a tombstone is and how this will be set / used as it is not obvious what this means.